<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phonebook</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Downloads/icon_image.png</normaloff>../../../../Downloads/icon_image.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(212, 209, 207, 1), stop:1 rgba(171, 208, 235, 1));


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="rowCount">
     <number>100</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Город</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Улица</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Дом</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Квартира</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 10pt &quot;Palatino Linotype&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

color: #873f2d;
font: 14pt &quot;Palatino Linotype&quot;;</string>
    </property>
    <property name="text">
     <string>Показать всех</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
